src/webgpu/capability_info.ts

```diff
@@ -269,6 +269,8 @@ export const kCanvasTextureFormats = ['bgra8unorm', 'rgba8unorm', 'rgba16float']
 // The alpha mode for canvas context.
 export const kCanvasAlphaModes = ['opaque', 'premultiplied'] as const;
 
+export const kCanvasColorSpaces = ['srgb', 'display-p3'] as PredefinedColorSpace[];
```

Comment:
done

Reply:
I'm not sure I understand the point of that. Is it just a place to add per enum info later?

Reply:
For obtuse typescript reasons, the type definition on those object tables forces them to have an entry for every single key in the key type. This way if something gets added to the `PredefinedColorSpace` string-literal-union type later on it (in a future tsc version), or to `GPUCanvasAlphaMode` (in a future webgpu types version), it will cause our code to fail typechecking so we know to update it.

There's probably a less obtuse way of achieving the same thing, but it's what we've used so far.